basics/how-storage-works/filecoin-plus.md

```diff
@@ -117,23 +116,26 @@ If making deals through the [API](https://github.com/filecoin-project/lotus/blob
 ]
 ```
 
+### How to Use DataCap, using the Command Line 
 If making deals from the command line, make sure to pass the flag `--verified-deal=true` as a parameter.
 
 ```shell
  lotus client deal --verified-deal=true
```

Comment:
the `client` subcommand no longer exists in the lotus cli, so this whole section should be removed.